### ISSUE TITLE
only about 0.5 val_acc

### ISSUE THREAD

**Issue by fiasco-zh** [fomorians/distracted-drivers-keras#4]
Hi, I have trained your model, but it is much over-fitting, more than **0.99 train-acc**, and about **0.55 val-acc**. Is this your final model and hy-parameters? I have trained some model architecture before, and there always be over-fitting even with using batch-norm, regularize and dropout. Do you have any advice?
